### DISCUSSION TITLE
How to manage ANTLR build configure with VCS

### DISCUSSION THREAD

**Discussion by oovm** [antlr/intellij-plugin-v4#662] [Q&A] [answered]
I want to know where this setting is stored

![image](https://github.com/antlr/intellij-plugin-v4/assets/17541209/07739b0b-b10c-4a4a-ae06-a7e1292a3777)

I want to synchronize this configuration through version control system like git, but I can't find where it is stored

**Comment by bjansen** [comment, marked as answer]
Hi, this kind of config is stored in `.idea/misc.xml` (this file can also contain stuff not related to ANTLR). There is currently no integration/synchronisation with Maven/Gradle builds.
